conf/cmi/language/sv/webform.webform.kasvatus_ja_koulutus_toiminta_av.yml

```diff
@@ -1,4 +1,4 @@
-title: 'Stadsstyrelsen, allmän bidragsansökan'
+title: 'Fostran och utbildning, ansökan om verksamhetsbidrag för organisatörer av eftermiddagsverksamhet för skolbarn'
 description: 'MVP-f&ouml;rm'
```

Comment:
Tän descriptionin voisi vielä napata täältä pois. Mä en tiedä mistä näitä edelleen valuu tänne (tai siis tiedän, kopioidaan miettimättä mitä kopioidaan)